### PR TITLE
Docs: What’s new & Upgrade guide & Breaking changes v11.0-preview

### PR DESCRIPTION
**Pull request by imatwawana**
**PLEASE DO NOT ADD DIRECTLY TO THIS PR**

- For What's new: Create entries using the website CMS
- For Upgrade guide and Breaking changes: Create a branch and make changes there for review prior to merging

************
This PR adds the What’s new, Upgrade guide, and Breaking changes for Grafana v11.0-preview. These same pages will be reused for Grafana v11.0.

**Note for reviewers**: Review for the What's new should be limited to placement of entries. Please do not make any copy edits directly except to the introductory text. To update entries, please update them in the What's new in Cloud entry and notify me.
*************

**TO DO**

- [ ] Add enablement videos as they are available - PR TBD
- [x] Update backport label
- [x] Merge and backport to v11.0 before April 8
- [ ] Backport to v10.4 April 8 - see details on PR #85699

### CODE REVIEW

**Review by Eve832 — APPROVED** [review]

File: docs/sources/breaking-changes/breaking-changes-v11-0.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+In these cases, we recommend that you take these steps before the upgrade to enable subfolders:
+
+- Create a copy of the affected routes and rewrite the matchers for the new copy. For example, if the original matcher was `grafana_folder=MyFolder/sub-folder`, then the new route matcher will be `grafana_folder=MyFolder\/sub-folder`.
+- After enabling subfolders, you can delete the old routes.
+
+Please note that if you use file provisioning, you can upgrade and update the routes at the same time.
+
+#### Learn more
+
+[Subfolders announcement](https://grafana.com/docs/grafana-cloud/whats-new/#subfolders)
```
Comment (jdbaldry) [inline docs/sources/breaking-changes/breaking-changes-v11-0.md:138]: This links to the Grafana Cloud post which is probably the same as this isn't it?

**Review by jdbaldry — APPROVED** [review]
Looks good to me, one usability nit

File: docs/sources/breaking-changes/breaking-changes-v11-0.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/breaking-changes/breaking-changes-v11-0.md:138]: Good catch, thank you!

Comment (jdbaldry) [reply docs/sources/breaking-changes/breaking-changes-v11-0.md:138]: Oh I just realized that this is the breaking changes doc which is different. I thought it was the What's new! 🤦

Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v11-0.md:138]:
```suggestion
```

Comment (jdbaldry) [reply docs/sources/breaking-changes/breaking-changes-v11-0.md]: Yeah I think that blog post does have more content. Maybe the link should be to the Grafana docs though?

Comment (imatwawana) [reply docs/sources/breaking-changes/breaking-changes-v11-0.md]: LOL! Whoops!

File: docs/sources/breaking-changes/breaking-changes-v11-0.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+In these cases, we recommend that you take these steps before the upgrade to enable subfolders:
+
+- Create a copy of the affected routes and rewrite the matchers for the new copy. For example, if the original matcher was `grafana_folder=MyFolder/sub-folder`, then the new route matcher will be `grafana_folder=MyFolder\/sub-folder`.
+- After enabling subfolders, you can delete the old routes.
+
+Please note that if you use file provisioning, you can upgrade and update the routes at the same time.
+
+#### Learn more
+
```
Comment (imatwawana) [inline docs/sources/breaking-changes/breaking-changes-v11-0.md:137]:
```suggestion
[Subfolders announcement](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/whatsnew/whats-new-in-v11-0/#subfolders)

```

**Comment by imatwawana** [comment]
Outstanding tasks on PR https://github.com/grafana/grafana/pull/85699

**Pull request merged**